tasks.py

```diff
@@ -17,78 +17,11 @@ def celery_main_task():
     items_list = orchestrator.get_items_data(db_queries=db_connection.get_db_queries())
 
     items_to_store = items_list[0]
-    out_of_stock_items = items_list[1]
 
-    orchestrator.out_of_stock_manger(db_queries=db_queries, user_utilities=user_utilities,
-                                     out_of_stock_items=out_of_stock_items)
+    if items_list[1] != []:
```

Comment:
There are nicer ways to check for empty list..
`if not items_list[1]` or something

Reply:
Didnt know if it will check for an empty list or for None, changed.